### PR TITLE
Docs example: Added tbody tag around tr in the data-table slots exemple

### PR DESCRIPTION
**Pull request by aspben**
Needed to render divider between lines correctly

## Motivation and Context

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x ] The PR title is no longer than 64 characters.
- [x ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/examples/data-tables/slots.vue [hunk 1]
```diff
@@ -52,12 +52,14 @@
 
       <template v-if="isEnabled('body')" v-slot:body="{ items }">
         <tr v-for="item in items" :key="item.name">
-          <td>{{ item.name }}</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
+          <tbody>
```
Comment (jacekkarczmarczyk) [inline packages/docs/src/examples/data-tables/slots.vue]: `tr > tbody > td` doesn't look like a correct markup

Comment (aspben) [reply packages/docs/src/examples/data-tables/slots.vue]: Sorry, now it's correct

File: packages/docs/src/examples/data-tables/slots.vue [hunk 2]
```diff
@@ -51,14 +51,16 @@
       </template>
 
       <template v-if="isEnabled('body')" v-slot:body="{ items }">
-        <tr v-for="item in items" :key="item.name">
+      <tbody>
```
Comment (sh7dm) [inline packages/docs/src/examples/data-tables/slots.vue]: add +2 spaces indentation to this block

**Comment by KaelWD** [comment]
Resolved by 09bfd7a40

**Pull request closed without merging**